### PR TITLE
Fix/set acti func in rnn base layer

### PR DESCRIPTION
**Pull request by lhs8928**
Commit 1: [layer] bugfix in set acti_func of rnn, lstm, gru
 - Set acti_func, recurrent_acti_func in finalize

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

Commit 2: [ahub] Init uninit member variable
 - Initialize uninit member variable

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

Commit 3: [unittest] Enable model_test_save_load_compare unittest
 - Except addition_resnet_like, multiple_output_model which contain
   multiout layer cause cannot assure the order of layer

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

**Review by zhoonit — APPROVED** [review]
LGTM :)

File: test/unittest/unittest_nntrainer_models.cpp [hunk 1]
```diff
@@ -1437,30 +1440,30 @@ INSTANTIATE_TEST_CASE_P(
       mkModelTc(preprocess_flip_validate, "3:1:1:10", 10, ModelTestOption::MINIMUM),
 
       /**< Addition test */
-      mkModelTc(addition_resnet_like, "3:1:1:10", 10, ModelTestOption::COMPARE),
+      mkModelTc(addition_resnet_like, "3:1:1:10", 10, ModelTestOption::COMPARE), // Enable option to ALL
```
Comment (kparichay) [inline test/unittest/unittest_nntrainer_models.cpp:1443]: TODO: Enable option to ALL?

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**